hw/top_darjeeling/lint/top_darjeeling_fpv_lint_cfgs.hjson

```diff
@@ -0,0 +1,153 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow: lint
+
+  // This is the primary cfg hjson for FPV linting. It imports ALL individual lint
+  // cfgs of the IPs FPV environments.
+  // This enables to run them all as a regression in one shot.
+  name: top_darjeeling_fpv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
+  // Different dashboard output path for each tool
+  rel_path: "hw/top_darjeeling/fpv/lint/{tool}/summary"
+
+  use_cfgs: [{
+              name: alert_handler_esc_timer_fpv
+              fusesoc_core: lowrisc:opentitan:top_darjeeling_alert_handler_esc_timer_fpv
```

Comment:
Hmm: do we have any way to do this yet? I think the Earlgrey version comes from `hw/top_earlgrey/ip_autogen/alert_handler/fpv`. I think this will probably need commenting until the Darjeeling equivalent lands?

Similarly for the ping timer.